test/functionalTest/cases/4330_onlyChangedAttrs_built_in_metadata/onlychangedattrs_not_working_with_built_in_metadata.test

```diff
@@ -0,0 +1,226 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+onlyChangedAttrrs not working with csubs cache
+
+--SHELL-INIT--
+dbDrop ${CB_DB_NAME} tk
+brokerStart CB 0 IPv4 -subCacheIval 2 -multiservice
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. Create entity E1 with attributes A and B
+# 02. Create subscription with onlyChangedAttrs true covering the entity
+# 03. Update E1-A
+# 04. Sleep until cache refres, update E1-A again
+# 05. Dump accumulator and see only A atribute in both notifications
+#
+
+
+echo '01. Create entity E1 with attributes A and B'
+echo '============================================'
+payload='{
+   "id": "E1",
+   "type": "T",
+   "A": {
+     "value": "foo",
+     "type": "Text"
+   },
+   "B": {
+     "value": 123,
+     "type": "Number"
+   }
+}'
+orionCurl --url /v2/entities --payload "$payload" --tenant tk --servicePath /
+echo
+echo
+
+
+echo '02. Create subscription with onlyChangedAttrs true and special metadata'
+echo '======================================================================'
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "idPattern": ".*"
+      }
+    ]
+   },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+    "onlyChangedAttrs":true,
+    "metadata": [ "previousValue" ],
+    "throttling": 0
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload" --tenant tk --servicePath /
+echo
+echo
+
+sleep 2s
```

Comment:
Why this sleep is needed? Have you observerd test glitching if not included?

Reply:
If removed, the sleep 2.5s statement in line 101 should be adjusted (to 4.5 I guess)

Reply:
I almost copied an existing test. I did not change that value, so I am not aware about test glitching if not included. Should I remove it? This does not hurt

Reply:
For the sake of the test simplicity, I'd suggest to remove it (adjusting to 4.5s in line 101).

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4335/commits/4be9bdbad5d7c9722d6524d1cb19f843fbed6922